### PR TITLE
Fix/deleted source event bug mockery

### PR DESCRIPTION
**Pull request by alxndr13**
builds upon #67 

cc: @HannesHil

### CODE REVIEW

File: Makefile
Comment (MichaelEischer) [inline Makefile:52]: That only works if `~/go/bin` is part of the users path. `$(go env GOPATH)/bin/mockery` would be more reliable.

But I'd prefer to not make changes to a users environment. `go run github.com/vektra/mockery/v2@v2.34.2` would also work, but takes half a second longer.

**Review by MichaelEischer — COMMENTED** [review]
The idea is to merge this PR once #67 is completed? Or should we rather treat them independently? (which would require a bit of rebasing to handle the regenerated mocks)

**Comment by alxndr13** [comment]
> The idea is to merge this PR once #67 is completed? Or should we rather treat them independently? (which would require a bit of rebasing to handle the regenerated mocks)

yes, after #67 is merged.

File: Makefile
Comment (alxndr13) [reply Makefile]: fixed.

**Comment by MichaelEischer** [comment]
@alxndr13 Running `make test` results in quite some changes in the generated mocks for me. Did you regenerate them after rebasing this PR to reflect the changes from #67?

**Comment by alxndr13** [comment]
> @alxndr13 Running `make test` results in quite some changes in the generated mocks for me. Did you regenerate them after rebasing this PR to reflect the changes from #67?

i believe my rebase went wrong. Will update the PR with fresh mocks.

**Review by MichaelEischer — APPROVED** [review]
@alxndr13 That seems to have done the trick.

**Pull request merged**
